bundle/compliance/cis_aws/rules/cis_1_19/test.rego

```diff
@@ -0,0 +1,44 @@
+package compliance.cis_aws.rules.cis_1_19
+
+import data.compliance.cis_aws.data_adapter
+import data.lib.test
+
+generate_certificate_resource(certificates) = {
+	"subType": "aws-iam-server-certificate",
+	"resource": {"certificates": certificates},
+}
+
+generate_expiration(expiration) = {"Expiration": expiration}
+
+last_year = time.format(time.add_date(time.now_ns(), -1, 0, 0))
+
+next_year = time.format(time.add_date(time.now_ns(), 1, 0, 0))
+
+test_violation {
+	eval_fail with input as generate_certificate_resource([generate_expiration(last_year)])
+	eval_fail with input as generate_certificate_resource([
+		generate_expiration(last_year),
+		generate_expiration(next_year),
+	])
+}
```

Comment:
fails when an expired certificate exists

Reply:
add it as a comment please